Packs/Microsoft365Defender/Classifiers/classifier-Microsoft_365_Defender_-_Incoming_Mapper.json

```diff
@@ -0,0 +1,344 @@
+                            }
+                        ]
+                    },
+                    "simple": ""
+                },
+                "Microsoft 365 Defender Classification": {
+                    "complex": null,
+                    "simple": "classification"
+                },
+                "Microsoft 365 Defender Devices": {
+                    "complex": {
+                        "accessor": "",
+                        "filters": [],
+                        "root": "Devices",
+                        "transformers": [
+                            {
+                                "args": {},
+                                "operator": "ConvertKeysToTableFieldFormat"
+                            }
+                        ]
+                    },
+                    "simple": ""
+                },
+                "Microsoft 365 Defender Display Name": {
+                    "complex": null,
+                    "simple": "incidentName"
+                },
+                "Microsoft 365 Defender First activity": {
+                    "complex": null,
+                    "simple": "First activity"
+                },
+                "Microsoft 365 Defender ID": {
+                    "complex": null,
+                    "simple": "incidentId"
+                },
+                "Microsoft 365 Defender Last activity": {
+                    "complex": null,
+                    "simple": "Last activity"
+                },
+                "Microsoft 365 Defender Status": {
+                    "complex": null,
+                    "simple": "status"
+                },
+                "Microsoft 365 Defender Tags": {
+                    "complex": {
+                        "accessor": "",
+                        "filters": [],
+                        "root": "tags",
+                        "transformers": []
+                    },
+                    "simple": ""
+                },
+                "Microsoft 365 Defender Users": {
+                    "complex": {
+                        "accessor": "",
+                        "filters": [],
+                        "root": "Impacted entities",
+                        "transformers": [
+                            {
+                                "args": {
+                                    "separator": {
+                                        "isContext": false,
+                                        "value": {
+                                            "complex": null,
+                                            "simple": ","
+                                        }
+                                    }
+                                },
+                                "operator": "join"
+                            },
+                            {
+                                "args": {},
+                                "operator": "3b758874-5a4b-417f-8ad8-5add2f5d8608"
```

Comment:
I'm guessing this should be `UserListToTable`

Reply:
right